website/models.py

```diff
@@ -103,4 +103,8 @@ class NotificationZone(models.Model):
     radius = models.DecimalField(max_digits=9, decimal_places=2)
 
     def __str__(self):
-        return self.name
\ No newline at end of file
+        return self.name
+
+class User(models.Model):
```

Comment:
Why are you creating a new user model? We are currently using the default django user model for everything so far.